snprc_ehr/resources/source_queries/create_v_snd_attributeData.sql

```diff
@@ -37,26 +34,26 @@ AS
 -- 10/11/2018 removing rows that have attribute values that are '' or null. tjh
 -- 05/13/19  Added a REPLACE to numeric/decimal CAST
 --           Purpose is to handle numeric data with commas (',') ~line 59  srr
+-- 04/23/2024 Lookup values need to be string values by default. tjh
 -- ==========================================================================================
 SELECT TOP (99.999999999) PERCENT
-  cp.ANIMAL_EVENT_ID               AS EventId,
-  sp.PKG_ID                        AS PkgId,
-  -- snd.EventData columns
-  cp.PROC_ID                       AS EventDataId,
-  sp.SUPER_PKG_ID                  AS SuperPkgId,
-  pbi.SUPER_PKG_ID                 AS ParentSuperPkgId,
-  cpa.value AS value,
+    cp.ANIMAL_EVENT_ID               AS EventId,
+        sp.PKG_ID                        AS PkgId,
+       -- snd.EventData columns
+       cp.PROC_ID                       AS EventDataId,
+       sp.SUPER_PKG_ID                  AS SuperPkgId,
+       pbi.SUPER_PKG_ID                 AS ParentSuperPkgId,
+       cpa.value AS value,
   -- exp.ObjectProperty columns
-  ltrim(rtrim(cpa.ATTRIB_KEY)) AS [KEY],
+  LTRIM(RTRIM(cpa.ATTRIB_KEY)) AS [_KEY],
 
-  CASE WHEN (LOWER(pa.DATA_TYPE) = 'numeric' OR
-    LOWER(pa.DATA_TYPE) = 'decimal')
+  CASE WHEN ( (LOWER(pa.DATA_TYPE) = 'numeric' OR LOWER(pa.DATA_TYPE) = 'decimal') ) AND pa.LOOKUP_KEY IS NULL
```

Comment:
Hi Terry. i think the check here needs to be pa.LOOKUP_KEY IS NOT NULL

Reply:
No - It is correct.

Reply:
If the lookup key exists, it needs to be placed in the StringValue column.